### DISCUSSION TITLE
custom query parameters panel missing or moved?

### DISCUSSION THREAD

**Discussion by samg666** [Budibase/budibase#3622] [Help]
Hello,

I have just upgraded from 0.9.173 to 1.0.2, and found the custom query params panel missing or moved to another place? 

BTW, why there's no release notice of the 1.0.2? What has been added and changed? 

![image](https://user-images.githubusercontent.com/29649651/144256461-4b4ea500-20fe-4468-8107-c99088e80fea.png)

**Comment by samg666** [comment]
Hi @mjashanks  Can you help on this? I couldn't anywhere to set the custom query parameters setting panel.

![image](https://user-images.githubusercontent.com/29649651/144344523-a978e289-65f5-431b-a74a-ef225042ce42.png)

**Comment by samg666** [comment]
The parameters panel was come back after downgraded to 0.9.189

![image](https://user-images.githubusercontent.com/29649651/144355694-54887955-f8f0-4612-a4b7-8ba0d058d830.png)

**Comment by mjashanks** [comment]
Hey Sam. Thanks for reporting this, I can reproduce it and have logged it here: https://github.com/Budibase/budibase/issues/3634

We'll look at this ASAP.

**Reply by shogunpurple** [reply]
Just to add to this:
> BTW, why there's no release notice of the 1.0.2? What has been added and changed?

We don't generally put out the release notes to everyone until we have had a few days to live with the new release, and iron out any issues raised by the community (like this one!)

We then create a GH discussion with all the release notes detailing the new features that have been added and what has changed. This will be released soon - over the next few days.
